scripts/assign_workflow_receive_sample.py

```diff
@@ -15,16 +16,19 @@ def _run(self):
             sample = art.samples[0]
             if sample.udf.get("User Prepared Library") == "Yes":
                 artifacts_to_route_userprepared.add(art)
-            else:
-                artifacts_to_route_preseqlab.add(art)
+                sample.udf['SSQC Result'] = 'Passed'
+                sample.put()
+        else:
+            artifacts_to_route_preseqlab.add(art)
```

Comment:
This `else` was meant to be for the `if` statement not for the `for` loop?
Reason this still pass the test is because the `else` in a for loop execute at the end of the loop which matches the second and last artifact tested.